### ISSUE TITLE
Several characters in the alt text result in no alt text at all

### ISSUE THREAD

**Issue by OddDev** [writefreely/writefreely#649]
### Describe the bug
Having certain characters in the alt text (as far as I tested, these are `:`, `"`, `'`) makes the alt-text on images not render. I suppose it's a parsing error returning `undefined` or an empty string.

### Steps to reproduce (if necessary)
1. Mak a new post
2. Add an image `[My alt text. That's very "funny".](/path/to/image)`
3. Open the rendered post
4. The alt text ist missing
5. Add another image `[My alt text. That is very funny.](/path/to/image)`
6. The alt text is there

### Expected behavior
The alt text should render regardless of the given input as long as it is compliant with W3 standards. 
https://html.spec.whatwg.org/multipage/embedded-content.html#dom-img-alt

### Application configuration
 - **Single mode** 
 - **Database?** [write.as]
 - **Open registration?** [yes]
 - **Federation enabled?** [yes]

**Version or last commit**:

**Comment by Naln1** [comment]
I am also experiencing this behavior. It is very frustrating as someone who cares a lot about accessibility, because it makes writing accurate and concise alt-text very difficult.

**Comment by thebaer** [comment]
Thanks for reporting this.

I've looked into it, and narrowed the issue down to our HTML sanitization policy, which isn't currently allowing HTML entities inside the `alt` attribute of our rendered HTML. It's a one-line fix -- I'll try to submit that soon.
